sql/planbuilder/scope.go

```diff
@@ -36,18 +37,37 @@ type scope struct {
 	ctes map[string]*scope
 	// groupBy collects aggregation functions and inputs
 	groupBy *groupBy
+	// windowFuncs is a list of window functions in the current scope
+	windowFuncs []scopeColumn
+	windowDefs  map[string]*sql.WindowDefinition
 	// exprs collects unique expression ids for reference
 	exprs map[string]columnId
 }
 
 func (s *scope) resolveColumn(table, col string, checkParent bool) (scopeColumn, bool) {
+	var found scopeColumn
+	var ok bool
```

Comment:
No top-level `ok` bools

Figure out what concept this is actually expressing and give it that name